document/CoreFormat.mdk

```diff
@@ -528,7 +586,7 @@ The `#pragma` warning construct in C# is such a syntactic construct.
 
 ### `fixes` property { #result-fixes }
 
-A `result` object may contain a property names `fixes`, whose value is a JSON array of `fix` objects (&sect;[#fix]).
+A `result` object may contain a property names `fixes`, whose value is a JSON array of zero or more `fix` objects (&sect;[#fix]).
```

Comment:
@michaelcfanning No, I changed my mind. Only one company asked for `fixes`. We can't require every tool to emit an empty array for every issue it finds.